Makefile

```
@@ -80,14 +80,7 @@ endif
 	touch .state/docker-build
 
 build:
-	docker-compose build --build-arg IPYTHON=$(IPYTHON) --force-rm web
-	docker-compose build --force-rm worker
-	docker-compose build --force-rm static
-
-	# Mark this state so that the other target will known it's recently been
-	# rebuilt.
-	mkdir -p .state
-	touch .state/docker-build
+	@make .state/docker-build
```

```suggestion
	@${MAKE} .state/docker-build
```

This ensures that the right `make` binary is used.

---

👍